### ISSUE TITLE
Faster release builds in watch&deploy workflow with `opt-level = 0`?

### ISSUE THREAD

**Issue by Boscop** [koute/cargo-web#127]
Context: My use case is described here:
https://github.com/koute/cargo-web/issues/122#issuecomment-407179918

Since "debug builds on the wasm32-unknown-unknown are currently totally broken forcing a release build", and it's taking so long to build with the watch&deploy workflow, could I maybe speed up the release build with:
```toml
[profile.release]
opt-level = 0
```
?

Or is this the part that's "currently totally broken"? 
Btw: If not this, which aspect of release builds is the broken one? :)

---

I tried doing the above, but I got:
```
warning: profiles for the non root package will be ignored, specify profiles at the workspace root:
package:   D:\projects\myproject\frontend\Cargo.toml
workspace: D:\projects\myproject\Cargo.toml
```
But my frontend depends on other crates of the workspace that it shares with the backend (e.g. msg types for ws communication, and all their constituent types), so I'd have to pull the frontend crate out of this workspace into a separate project? 

I'd do that, but only if the above setting would speed up the release build..

**Comment by koute** [comment]
Unfortunately that's not going to work right now; the optimized builds are necessary for the `js!` macro to be functional.

However, I do plan to try to migrate the `js!` macro on `wasm32-unknown-unknown` to custom sections soon, so building in release mode should stop being necessary.

**Comment by Boscop** [comment]
Would `opt-level = 1` work? :)

**Comment by koute** [comment]
Probably not; but you can try if you want. (You probably have to modify `cargo-web` though as currently it enforces the optimization level through `RUSTFLAGS`, so it will override whatever you specify in your `Cargo.toml`.)

**Comment by koute** [comment]
Fixed in 0.6.22 where debug builds are supported on `wasm32-unknown-unknown`. Requires still unreleased `stdweb` though.
